tests/e2e/configurer/chain/node.go

```diff
@@ -143,3 +147,29 @@ func (n *NodeConfig) LogActionF(msg string, args ...interface{}) {
 	s := fmt.Sprintf(msg, args...)
 	n.t.Logf("[%s] %s. From container %s", timeSinceStart, s, n.Name)
 }
+
+func (n *NodeConfig) NodeStatus() error {
+	cmd := []string{"osmosisd", "status"}
+	_, errBuf, err := n.containerManager.ExecCmd(n.t, n.Name, cmd, "")
+	if err != nil {
+		return err
+	}
+
+	type validatorInfo struct {
+		Address     bytes.HexBytes
+		PubKey      cryptotypes.PubKey
+		VotingPower int64
+	}
+
+	type resultStatus struct {
+		NodeInfo      p2p.DefaultNodeInfo
+		SyncInfo      coretypes.SyncInfo
+		ValidatorInfo validatorInfo
```

Comment:
Does it make sense to return this status? Otherwise, why would we parse it? Maybe there is no use case right now but there might be in the future

Reply:
I found it in the SDK, it would be better if we unmarshal the response according to its struct.

Reply:
https://github.com/osmosis-labs/osmosis/pull/4150#discussion_r1091427246